.gitignore

```diff
@@ -0,0 +1,3 @@
+# Dependency directories
+node_modules/
+yarn.lock
```

Comment:
When does `yarn.lock` get created? On my system I am only getting `package-lock.json` generate which we should ignore too.

Reply:
`A `yarn.lock` file is only generated if someone runs `yarn install`

Likewise, a `package-lock.json` is only generated if you run `npm install`